tests/gpflow/optimizers/test_mcmc.py

```diff
@@ -34,12 +34,14 @@ def build_model(data):
     return model
 
 
-def build_model_with_uniform_prior(data, prior_on, prior_width):
+def build_model_with_uniform_prior_no_transforms(data, prior_on, prior_width):
     def parameter(value):
         low_value = -100
         high_value = low_value + prior_width
         prior = Uniform(low=np.float64(low_value), high=np.float64(high_value))
-        return gpflow.Parameter(value, prior=prior, prior_on=prior_on)
+        return gpflow.Parameter(
+            value, transform=tfp.bijectors.Identity(), prior=prior, prior_on=prior_on
```

Comment:
```suggestion
            value, prior=prior, prior_on=prior_on
```
shouldn't this test the setting of the default (identity) transform?

Reply:
When `value` is a `Parameter` and `transform=None`, the `value.transform` is used to set the transform of the new parameter. You are correct to say that this test is checking the identity transform, so to ensure that it is being used we set it directly in the test. Otherwise we have to assume that the incoming `Parameter` is using the identity transform which is not always true.